.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```

Comment:
this file should not be ignored, people should not commit files they did not make changes to intentionally